documentation/business-analyst-activities.md

```diff
@@ -1,19 +1,63 @@
 # Business Analyst Activities
 
-You must have a GitHub account and access to the Teaching Vacancies repository to be able to make any changes to the code base, or to edit/create files (e.g, .yml or .md text files)
-<Insert link here to new page> This file explains the process by which new team members can get the access they require on GitHub.
+You must have a GitHub account and access to the Teaching Vacancies repository to be able to make any changes to the code base.
 
-## Updating Polygon Mappings in the mapped_locations.yml file
+The [onboarding](../onboarding.md) document explains the process by which new team members can get the access they require on GitHub.
+
+## Markdown files
+
+We use [`.md` Markdown](https://www.markdownguide.org/cheat-sheet) to write this file and the others in the [documentation](./documentation) directory.
+
+Go to [HackMD](https://hackmd.cloudapps.digital/) which is hosted on Gov.UK PaaS, and click `New guest note`. You can then learn Markdown while writing a document:
+- enter some text, highlight it
+- press `B` to make it bold
+- press `H` to create headings (press `H` again to change from an H1 to an H2 heading)
+
+You'll see an instant preview of your document in the right-hand pane.
+
+## YAML files
+
+We use [`.yml` YAML](https://rollout.io/blog/yaml-tutorial-everything-you-need-get-started/) for data files.
+
+Go to [yamlchecker.com](https://yamlchecker.com/), and start typing YAML, e.g. the first two lines of [mapped_locations.yml](../lib/tasks/data/mapped_locations.yml):
+```
+---
+- ['barking', 'barking and dagenham']
+```
+You'll get instant feedback - either `Valid YAML!` or a useful error
```

Comment:
V useful - I presume, in addition to writing this, you've told our BAs?

Reply:
Thanks. I've been through it with Ben, and will "socialise" the information more widely on merging.